<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QCheckBox" name="m_PauseVoipCheckBox">
      <property name="text">
       <string>Pause VOIP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_PauseKodiCheckbox">
      <property name="text">
       <string>Pause Kodi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_MuteSpeakersCheckBox">
      <property name="text">
       <string>Mute Speakers</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_MuteMicrophoneCheckBox">
      <property name="text">
       <string>Mute Microphone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Waveform" name="m_WaveForm" native="true"/>
    </item>
    <item>
     <widget class="LogWidget" name="m_LogWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Waveform</class>
   <extends>QWidget</extends>
   <header location="global">QtSoundTest/WaveForm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">LogWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
